playground/playground.py

```diff
@@ -0,0 +1,43 @@
+import json
+import re
+from fnmatch import fnmatch
+
+from ply import lex
+from redbot.core import checks
+from redbot.core import commands
+from redbot.core.utils.chat_formatting import box, pagify
+
+from rpadutils import rpadutils
+
+class Playground(commands.Cog):
+    """Just lil' fun things."""
+
+    def __init__(self, bot, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.bot = bot
+
+    @commands.command(aliases = ["parrot", "repeat"])
```

Comment:
I would like you to pause for a moment and consider the impact of adding a function that lets anyone make the bot say arbitrary things.

Please either:
1) Add manage_messages permission as a restriction
2) If you're feeling ambitious, make this per-server toggleable to public use (with the default private to mods).

Reply:
oh i forgot to add that sorry, i swear i was gonna